include/umf/memory_pool.h

```diff
@@ -19,8 +20,6 @@ extern "C" {
 
 typedef struct umf_memory_pool_t *umf_memory_pool_handle_t;
 
-struct umf_memory_pool_ops_t;
```

Comment:
why you replaced forward declaration of the `struct umf_memory_pool_ops_t;` with the `#include <umf/memory_pool_ops.h>`

Reply:
By mistake. Reverted. Thanks.